code/game/objects/items/weapons/storage/wallets.dm

```diff
@@ -122,3 +117,86 @@
 		new item2_type(src)
 	if(item3_type)
 		new item3_type(src)
+
+/obj/item/weapon/storage/wallet/proc/mob_icon_update()
+	if (ismob(src.loc))
+		var/mob/M = src.loc
+		M.update_inv_wear_id()
+
+/obj/item/weapon/storage/wallet/verb/flip_side()
```

Comment:
usr to user

Reply:
Going to need a bit of reworking; won't take user as a src setting if you want me to do ALL of them.